Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM python:3.7-slim
+FROM debian:bullseye
```

Comment:
```suggestion
FROM debian:bullseye-slim
```

Is it possible to use `bullseye-slim` instead?

Reply:
Yes, I just tested with bullseye-slim and this also seems to work.